src/components/pages/dash-wrapper/DashWrapper.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
This my friend, is an awesome extrapolation of functionality. I think this is beautifully done as it helps keep our components modular and reusable. I hadn't thought of building it this way and I think this is such a cool way to do this. I will definitely remember this and use this in future projects!